src/App.tsx

```diff
@@ -25,29 +27,39 @@ interface AppProps extends RouteComponentProps {
 }
 
 const App: FunctionComponent<AppProps> = ({ history, location, loginState }) => {
-	const PRIMARY_ITEMS: NavigationItem[] = [
-		{ label: 'Home', location: APP_PATH.HOME },
-		{
-			label: 'Zoeken',
-			location: APP_PATH.SEARCH,
-			icon: 'search',
-		},
-		{ label: 'Ontdek', location: APP_PATH.DISCOVER },
-		{
-			label: 'Mijn Werkruimte',
-			location: APP_PATH.WORKSPACE,
-			icon: 'briefcase',
-		},
-		{ label: 'Projecten', location: APP_PATH.PROJECTS },
-		{ label: 'Nieuws', location: APP_PATH.NEWS },
-	];
-	const SECONDARY_ITEMS: NavigationItem[] =
-		loginState && loginState.message === LoginMessage.LOGGED_IN
-			? [{ label: 'Afmelden', location: APP_PATH.LOGOUT }]
-			: [
-					{ label: 'Registreren', location: APP_PATH.PUPIL_OR_TEACHER },
-					{ label: 'Aanmelden', location: APP_PATH.REGISTER_OR_LOGIN },
-			  ];
+	const isLoggedIn = loginState && loginState.message === LoginMessage.LOGGED_IN;
+	let PRIMARY_ITEMS: NavigationItem[];
+	let SECONDARY_ITEMS: NavigationItem[];
+	if (isLoggedIn) {
+		PRIMARY_ITEMS = [
+			{ label: 'Home', location: APP_PATH.LOGGED_IN_HOME },
+			{
+				label: 'Zoeken',
+				location: APP_PATH.SEARCH,
+				icon: 'search' as IconName,
```

Comment:
Die `as IconName` moet hier niet expliciet, dat wordt opgevangen in het `NavigationItem` type